web/.env

```
@@ -0,0 +1,6 @@
+VITE_FIREBASE_API_KEY=AIzaSyCk27mGASnumYCTeMRpPjjakg89MhgaAQU
+VITE_FIREBASE_AUTH_DOMAIN=carwatch-5cabf.firebaseapp.com
+VITE_FIREBASE_PROJECT_ID=carwatch-5cabf
+VITE_FIREBASE_STORAGE_BUCKET=carwatch-5cabf.firebasestorage.app
+VITE_FIREBASE_MESSAGING_SENDER_ID=1066853456315
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FIREBASE_MESSAGING_SENDER_ID key should go before the VITE_FIREBASE_PROJECT_ID key

(UnorderedKey)